app/views/home/index.html.slim

```diff
@@ -1 +1,23 @@
-h1 New Vasjuky Project
+  .jumbotron.baner
+    h1 Hello, Франківськ!
+    p.light До вашої уваги соцаліний проект під кодовою назвою "NewVasjuky" групи IF-070.WebUI-RoR,
+            який має на меті #{'#{ довга мотиваційна стрінга }'}...
+
+    .button.btn.btn-primary.btn-lg.center
+        = link_to 'Повідомити про проблему', new_issue_path
+  ul.nav.nav-pills role="tablist"
+    li role="presentation"
+        span.badge #{@count}
+        span.phrase Проблем додано
+    li role="presentation"
+        span.badge #{@approved_count}
+        span.phrase Проблем вирішено
+  .row
+    h3.indent Останні додані:

```

Comment:
remove this class "indent" completely

inside ROW you MUST put content inside COL element. and you will not need this class.

this is the rule!